src/_includes/docs/add-to-app/ios-project/embed-frameworks.md

```diff
@@ -0,0 +1,117 @@
+### Embed frameworks in Xcode {:#method-b .no_toc}
+
+#### Approach {:#method-b-approach}
+
+In this second method, edit your existing Xcode project,
+generate the necessary frameworks, and embed them in your app. You need to create three iOS frameworks: for the Flutter engine, for your compiled Dart code, and for all Flutter plugins in Xcode. Embed these frameworks and update your existing application's build settings.
```

Comment:
Nit: we create a framework for each plugin, so your app can have more than 3 frameworks.